### ISSUE TITLE
Staging Review finding: Loading indicator is not communicated to assistive technology. 

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#128358]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Appointments Team
**Product:** Appointments
**Feature:** Oracle Health Scheduling - Direct Schedule and Request for an Appointment

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

When loading indicator is onscreen, the only announcement to a screen reader user is "new appointment". The other text, "Finding available locations for your appointment" is not announced, nor is the loading state.  

Other occurences:
- Loading components throughout the flow

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="294" height="181" alt="Loading" src="https://github.com/user-attachments/assets/a4362ea3-3b7c-455c-89e0-4f242942f8a3" />

</details>

## Recommended action

Ensure to follow VADS guidance, https://design.va.gov/components/loading-indicator, to ensure that the loading indicator is announced correctly.

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** [WCAG 4.1.3 Status Messages - Level AA](https://www.w3.org/TR/WCAG22/#status-messages)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by eselkin** [comment]
This only affects JAWS (not Mac VoiceOver or Windows NVDA)

It seems this may be an issue to bring up to VADS because the loading indicator is the VADS component with the label and the set-focus prop set to true. We don't have control on how to make it more assertive. We can wrap the component in a div with role alert and that may announce it but that may not be acceptable use of divs but it doesn't seem likely.

**Comment by eselkin** [comment]
As a side note, I tried all aria-live possibilities in conjunction with aria-atomic="true" but none worked. It seems odd because role="alert" should be the same as aria-live="assertive" and aria-atomic="true" but it is not. Maybe something about the component catches one of those and prevents it internally.

**Comment by eselkin** [comment]
As there was no response from DST on the issue, I am merging this now.

**Comment by eselkin** [comment]
Also, if whoever tests this wants to pair with me so I can show it to them on JAWS on windows, I'd be happy to help.
